recipes/emsdk/all/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class EmSDKConan(ConanFile):
+    name = "emsdk"
+    description = "Emscripten is an Open Source LLVM to JavaScript compiler"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/kripken/emscripten"
+    topics = ("conan", "emsdk", "emscripten", "sdk")
+    license = "MIT"
+    settings = "os"
```

Comment:
Probably we need `arch` here too. We are now building Apple M1 as well (`MacOS` + `armv8`). I don't now if there are working binaries for it, but probably regular MacOS ones won't work, right?

Reply:
I do not know about MacOs.